core/base/src/main/java/alluxio/Constants.java

```diff
@@ -194,6 +194,8 @@
   public static final int REPLICATION_MAX_INFINITY = -1;
 
   // Persistence
+  // The file should only be persisted after rename operation or persist CLI
+  public static final int NO_AUTO_PERSIST_VALUE = -1;
```

Comment:
nit: I think `_VALUE` is redundant in naming? Can we just call it `NO_AUTO_PERSIST`?

Reply:
Removed `_value`